src/Nox.Types.Abstractions/TypeOptions/VatNumberOptions.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Nox.Types;
+
+public class VatNumberOptions : INoxTypeOptions
+{
+    public string DefaultCountryCode { get; internal set; } = "GB";
```

Comment:
we are following a convention like this:

```
    public static readonly string DefaultCountryCode = "GB";
    public string CountryCode{ get; set; } = DefaultCountryCode ;
```

Reply:
Have updated the code to follow the above mentioned convention